QUICKSTART.md

```diff
@@ -0,0 +1,45 @@
+# Kind
+This guide was created to help new users of Kubeflow install and test Kubeflow Helm Charts locally based on the kind.
```

Comment:
We provide multiple quickstart scripts, I think it would be much better to explain all of the quickstarts. Is this something you could add?

Some other resources you might find helpful in describing other quickstarts:
* https://kromanow94.github.io/kubeflow-manifests/ (it's pretty old but was first explanation of the quickstart scripts and the initiative)
* https://github.com/kubeflow/kubeflow/issues/3173#issuecomment-2015462968
* https://github.com/kubeflow/kubeflow/issues/3173#issuecomment-2039430994
* https://github.com/kubeflow/kubeflow/issues/3173#issuecomment-2103150329
* https://github.com/kubeflow/manifests/issues/2730